contracts/pods/PodManager.sol

```diff
@@ -0,0 +1,136 @@
+        address _podAdmin,
+        address _podController,
+        address _memberToken
+    ) {
+        require(_core != address(0), "Zero address");
+        require(_podAdmin != address(0x0), "Zero address");
+        require(_podController != address(0x0), "Zero address");
+
+        core = _core;
+        podAdmin = _podAdmin;
+        podController = IControllerV1(_podController);
+        memberToken = IMemberToken(_memberToken);
+    }
+
+    ///////////////////// GETTERS ///////////////////////
+
+    /// @notice Get the address of the Gnosis safe that represents a pod
+    /// @param podId Unique id for the orca pod
+    function getPodSafe(uint256 podId) external view returns (address) {
+        return podController.podIdToSafe(podId);
+    }
+
+    //////////////////// STATE-CHANGING API ////////////////////
+
+    /// @notice Create a child Orca pod. Callable by the DAO and the Tribal Council
+    /// @param _members List of members to be added to the pod
+    /// @param _threshold Number of members that need to approve a transaction on the Gnosis safe
+    /// @param _podLabel Metadata, Human readable label for the pod
+    /// @param _ensString Metadata, ENS name of the pod
+    /// @param _imageUrl Metadata, URL to a image to represent the pod in frontends
+    /// @param minDelay Delay on the timelock
+    function createChildOptimisticPod(
+        address[] memory _members,
+        uint256 _threshold,
+        bytes32 _podLabel,
+        string memory _ensString,
+        string memory _imageUrl,
+        uint256 minDelay
+    ) public onlyPodAdmin returns (uint256, address) {
+        uint256 podId = memberToken.getNextAvailablePodId();
+
+        podController.createPod(
+            _members,
+            _threshold,
+            podAdmin,
+            _podLabel,
+            _ensString,
+            podId,
+            _imageUrl
+        );
+        address safeAddress = podController.podIdToSafe(podId);
+        address timelock = createOptimisticTimelock(
+            core,
+            safeAddress, // proposer
+            safeAddress, // executor
+            minDelay
+        );
+
+        // Set mapping from podId to timelock for reference
+        getPodTimelock[podId] = timelock;
+
+        emit CreatePod(podId, safeAddress);
+        return (podId, timelock);
+    }
+
+    /// @notice Create an Optimistic timelock, with a proposer and executor
+    /// @param _core Fei core address
+    /// @param proposer Timelock proposer address
+    /// @param executor Timelock executor address
+    /// @param minDelay Delay on the timelock before execution
+    function createOptimisticTimelock(
+        address _core,
+        address proposer,
+        address executor,
+        uint256 minDelay
+    ) internal returns (address) {
+        address[] memory proposers = new address[](1);
+        proposers[0] = proposer;
+
+        address[] memory executors = new address[](1);
+        executors[0] = executor;
```

Comment:
execution can be basically public or abstracted away. I think it could be cool to create a single "executor" contract which has the access granted upon timelock creation and exposing execution publicly.

Maybe even worth considering a public proposer and vetoer contract. Essentially porting the access control to other fei contracts and logic.